pkg/synopsysctl/soperator_resources.go

```diff
@@ -274,3 +277,86 @@ func (specConfig *SOperatorSpecConfig) GetOperatorClusterRoleBinding() *horizonc
 
 	return synopsysOperatorClusterRoleBinding
 }
+
+// GetOperatorClusterRole creates a ClusterRole Horizon component for the Synopsys-Operaotor
+func (specConfig *SOperatorSpecConfig) GetOperatorClusterRole() *horizoncomponents.ClusterRole {
+	synopsysOperatorClusterRole := horizoncomponents.NewClusterRole(horizonapi.ClusterRoleConfig{
+		APIVersion: "rbac.authorization.k8s.io/v1beta1",
+		//ClusterName : "string,"
+		Name:      "synopsys-operator-admin",
+		Namespace: specConfig.Namespace,
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{"apiextensions.k8s.io"},
+		Resources:       []string{"customresourcedefinitions"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{"rbac.authorization.k8s.io"},
+		Resources:       []string{"clusterrolebindings", "clusterroles"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+		APIGroups:       []string{""},
+		Resources:       []string{"namespaces", "configmaps", "persistentvolumeclaims", "services", "secrets", "replicationcontrollers", "deployments", "statefulsets", "serviceaccounts"},
+		ResourceNames:   []string{},
+		NonResourceURLs: []string{},
+	})
+
+	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
+		Verbs:           []string{"get", "list", "watch"},
+		APIGroups:       []string{""},
+		Resources:       []string{"pods", "pods/log", "endpoints"},
```

Comment:
Move pods into the previous resource list.  We'll want to be able to do all those things to pods as well if a pod is needed to be deployed standalone.

Reply:
Moved them into the previous resource list (which is now a *).